dashboard/app/models/script_level.rb

```diff
@@ -133,9 +133,30 @@ def summarize
     summary
   end
 
+  SCRIPT_LEVEL_CACHE_KEY = 'script-level-cache'
+
+  def self.script_level_cache_to_cache
+    Rails.cache.write(SCRIPT_LEVEL_CACHE_KEY, script_level_cache_from_db)
+  end
+
+  def self.script_level_cache_from_cache
+    ScriptLevel.connection
+    [ScriptLevel, Level, Game, Concept, Callout, Video,
+     Artist, Blockly].each(&:new) # make sure all possible loaded objects are completely loaded
+    Rails.cache.read SCRIPT_LEVEL_CACHE_KEY
+  end
+
+  def self.script_level_cache_from_db
+    ScriptLevel.includes([{level: [:game, :concepts]}, :script]).index_by(&:id)
```

Comment:
the includes for the script levels in the Script cache are:
````
script_levels: [{level: [:game, :concepts] }, :stage, :callouts]}
````

Should these be the same? I think I saw some queries on the stages table in your log that might have been cause by not preloading the stage association. For that matter, if do want them to be identical, we could just build the script level cache with the script levels from the Script cache at the time that we load that. And this would just be rearranging objects we have in memory instead of querying the db again for the same data.